src/components/ui/Button.js

```diff
@@ -21,7 +21,7 @@ export function IconButton ({ onClick, children, active, text, className }) {
 
 export const FilterToggleButton = ({ selected, label, onClick }) => {
   return (
-    <button type='button' onClick={onClick} className={`border-2 rounded-2xl  btn-small active:scale-95 ${selected ? 'border-neutral-100 text-neutral-100' : 'border-neutral-500 text-neutral-400 hover:border-neutral-400'}`}>
+    <button type='button' onClick={onClick} className={`border-2 rounded-2xl  btn-small active:scale-95 ${selected ? 'border-neutral-800 text-neutral-800' : 'border-neutral-400 text-neutral-400 hover:border-neutral-600'}`}>
       {label}
     </button>
   )
```

Comment:
why are the other files tsx? but Button is .js?

Reply:
we migrated to TS 2 months ago and not everything was ported